docs/ingest-data/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Ingest data from multiple sources
+---
+
+import DocCardList from '@theme/DocCardList';
+
+<DocCardList />
```

Comment:
It's a react component that renders one div block per ingest tutorial (see screenshot).